File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/DocumentManifest.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.Resource;

public class DocumentManifest : DomainResource, IResource
{
    public string? Description { get; set; }
    public required Base.ResourceReference[] Content { get; set; }
    public Base.ResourceReference[]? Recipient { get; set; }
    public Base.CodeableConcept? Type { get; set; }
    public string? Created { get; set; }
    public DocumentManifest_Related[]? Related { get; set; }
    public string? Source { get; set; }
    public Base.ResourceReference[]? Author { get; set; }
    public Base.Identifier? MasterIdentifier { get; set; }
    public required string Status { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
    public Base.ResourceReference? Subject { get; set; }

    public class DocumentManifest_Related : BackboneElement
    {
        public Base.ResourceReference? Ref { get; set; }
        public Base.Identifier? Identifier { get; set; }
    }
}

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/DocumentManifest.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import CodeableConcept
from base import DomainResource
from base import Reference
from base import Identifier
from base import BackboneElement

@dataclass(kw_only=True)
class DocumentManifestRelated(BackboneElement):
    ref: Optional[Reference] = None
    identifier: Optional[Identifier] = None

@dataclass(kw_only=True)
class DocumentManifest(DomainResource):
    description: Optional[str] = None
    content: list[Reference] = field(default_factory=list)
    recipient: Optional[List[Reference]] = field(default_factory=list)
    type: Optional[CodeableConcept] = None
    created: Optional[str] = None
    related: Optional[List[DocumentManifestRelated]] = field(default_factory=list)
    source: Optional[str] = None
    author: Optional[List[Reference]] = field(default_factory=list)
    master_identifier: Optional[Identifier] = None
    status: str
    identifier: Optional[List[Identifier]] = field(default_factory=list)
    subject: Optional[Reference] = None